dialogue-core/src/main/java/com/palantir/dialogue/core/StickyChannelRejectedException.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.dialogue.core;
+
+import com.palantir.logsafe.exceptions.SafeRuntimeException;
+
+@SuppressWarnings("ImmutableEnumChecker")
+enum StickyChannelRejectedException {
+    INSTANCE;
+
+    private final RuntimeException exception;
+
+    StickyChannelRejectedException() {
+        this.exception = new SafeRuntimeException("Rejected");
```

Comment:
Stack trace will always point to the StickyChannelRejectedException class initialization location.
Potential memory leak if anyone calls `addSuppressed(Throwable)`. We can define a new SafeLoggable exception which overrides `fillInStackTrace()` and `addSuppressed(Throwable exception)` as no-op to prevent these issues.

Reply:
This was just silly Jakub, didn't realize I can just call permit#dropped directly, instead of having to wire this into the Behavior.